### DISCUSSION TITLE
Starting learn to  together " web3 security researcher "

### DISCUSSION THREAD

**Discussion by cysky0x1** [Cyfrin/foundry-full-course-cu#1536] [General]
Hello everyone,
 I wanted to wonder if anyone is on their way to becoming a web3 security researcher
And he started learning, 
I just finished the  lesson 0 and 1 in Course " Prerequisites "

We can encourage each other 

thanks all

**Comment by EngrPips** [comment]
Awesome, I wish you the best of luck on your journey buddy

**Reply by cysky0x1** [reply]
Hi @EngrPips 
How long did it take you to discover your first vulnerability?

**Reply by EngrPips** [reply]
Sadly i have only done one real audit and didn't have a valid findings. since then i haven't done another because i have list of stuffs lined up to be completed before i take on my next audit, some of those are 

=> study and understand uniswap codebase from (v1 -> v3)
=> study and understand aave coodebase from 
=> finish solidity documentation
=> finish foundry book and some more other stuffs

So right now i would say i am cooking and if you have found your first vulnerability already you are far better than i am and i should be learning from you.

**Reply by cysky0x1** [reply]
I found , but in the web 2 like xss ..... , but i started learning about web 3

**Reply by EngrPips** [reply]
not bad not bad

**Reply by EngrPips** [reply]
Thats fine buddy

**Reply by EngrPips** [reply]
Alright buddy

**Comment by ljb630** [comment]
Hey! discord me @ljb630web3

**Comment by ARomano79** [comment]
I am still learning Foundry 101, but I am looking forward to learning Security Research and Auditing.

**Reply by EngrPips** [reply]
One at a time and before you know it you have learnt it all !

**Comment by sharonphiliplima** [comment]
Discord - sharonphiliplima

**Reply by EngrPips** [reply]
awesome

**Reply by sharonphiliplima** [reply]
Where can I find you?

**Reply by EngrPips** [reply]
Discord - engr.pips
twitter - @EngrPips

**Comment by raduom** [comment]
I am also starting on the journey to become a smart contract auditor. My discord is @raduom if you want to get in touch. I am a reasonably decent software developer with a special interest in exotic programming languages.

**Reply by EngrPips** [reply]
Awesome, i wish you the best of luck on your journey of knowledge buddy
